src/EA.Weee.Web/Areas/Admin/Controllers/AddAatfController.cs

```diff
@@ -1,23 +1,32 @@
 ﻿namespace EA.Weee.Web.Areas.Admin.Controllers
 {
     using EA.Prsd.Core.Domain;
+    using EA.Prsd.Core.Extensions;
     using EA.Weee.Api.Client;
     using EA.Weee.Core.AatfReturn;
     using EA.Weee.Core.Organisations;
     using EA.Weee.Core.Search;
     using EA.Weee.Core.Shared;
     using EA.Weee.Requests.Admin;
     using EA.Weee.Requests.Organisations;
+    using EA.Weee.Requests.Organisations.Create;
+    using EA.Weee.Requests.Organisations.Create.Base;
     using EA.Weee.Requests.Shared;
+    using EA.Weee.Security;
     using EA.Weee.Web.Areas.Admin.Controllers.Base;
     using EA.Weee.Web.Areas.Admin.ViewModels.AddAatf;
+    using EA.Weee.Web.Areas.Admin.ViewModels.AddAatf.Details;
+    using EA.Weee.Web.Areas.Admin.ViewModels.AddAatf.Type;
+    using EA.Weee.Web.Constant;
+    using EA.Weee.Web.Filters;
     using EA.Weee.Web.Infrastructure;
     using System;
     using System.Collections.Generic;
     using System.Linq;
     using System.Threading.Tasks;
     using System.Web.Mvc;
 
+    [AuthorizeInternalClaimsAttribute(Claims.InternalAdmin)]
```

Comment:
Add unit test to check this attribute is on the controller

Reply:
Added but can't test if it has the Claims.InternalAdmin because it's a private member